lib/ansible/modules/packaging/os/rhn_register.py

```diff
@@ -35,7 +35,7 @@
 notes:
     - In order to register a system, rhnreg_ks requires either a username and password, or an activationkey.
 requirements:
-    - rhnreg_ks
+    - rhnreg_ks, either libxml2 or lxml
```

Comment:
Indeed, fixed in https://github.com/ansible/ansible/pull/26878/commits/b076c80cb208cf34c26a278d1544094cd52c63c2.